error G76A9B1F6: The method '_FluentTextSelectionControls.buildHandle' has fewer positional arguments than those of overridden method

**Describe the bug**
First tried to build with my application then tried to run the example one and I get the same exact error.

**To Reproduce**
Steps to reproduce the behavior:
1. Run the example project to windows
2. See error before build finishes

```
[√] Flutter (Channel beta, 2.5.0, on Microsoft Windows [Version 10.0.19043.1165], locale en-US)
    • Flutter version 2.5.0 at C:\Users\Elias\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4cc385b4b8 (6 days ago), 2021-09-07 23:01:49 -0700
    • Engine revision f0826da7ef
    • Dart version 2.14.0

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\Elias\AppData\Local\Android\sdk
    • Platform android-S, build-tools 30.0.3
    • ANDROID_HOME = C:\Users\Elias\AppData\Local\Android\sdk
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.9.4)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.9.31205.134
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2020.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[√] VS Code (version 1.60.0)
    • VS Code at C:\Users\Elias\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.26.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19043.1165]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 93.0.4577.63
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 93.0.961.47

• No issues found!
```

```
PS C:\Users\Elias\Downloads\fluent_ui-master\example> flutter run
Multiple devices found:
Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19043.1165]
Chrome (web)      • chrome  • web-javascript • Google Chrome 93.0.4577.63
Edge (web)        • edge    • web-javascript • Microsoft Edge 93.0.961.47
[1]: Windows (windows)
[2]: Chrome (chrome)
[3]: Edge (edge)
Please choose one (To quit, press "q/Q"): 1
Launching lib\main.dart on Windows in debug mode...
../lib/src/controls/form/selection_controls.dart(50,10): error G76A9B1F6: The method '_FluentTextSelectionControls.buildHandle' has fewer positional arguments than those of overridden method 'TextSelectionControls.buildHandle'. [C:\Users\Elias\Downloads\fluent_ui-master\example\build\windows\flutter\flutter_assemble.vcxproj]
../lib/src/controls/form/selection_controls.dart(61,10): error G76A9B1F6: The method '_FluentTextSelectionControls.getHandleAnchor' has fewer 
positional arguments than those of overridden method 'TextSelectionControls.getHandleAnchor'. [C:\Users\Elias\Downloads\fluent_ui-master\example\build\windows\flutter\flutter_assemble.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(240,5): error MSB8066: Custom build for 'C:\Users\Elias\Downloads\fluent_ui-master\example\build\windows\CMakeFiles\d68200f278100e31c8ef127274f802e5\flutter_windows.dll.rule;C:\Users\Elias\Downloads\fluent_ui-master\example\build\windows\CMakeFiles\3ded2e916a2acf0c7491b781f0d55533\flutter_assemble.rule' exited with code 1. [C:\Users\Elias\Downloads\fluent_ui-master\example\build\windows\flutter\flutter_assemble.vcxproj]
Building Windows application...
Exception: Build process failed.
```

@bdlukaa I get the same error in the stable channel after a flutter clean.

```
PS C:\Users\Elias\Documents\GitHub\subathontimer> flutter run
Multiple devices found:
Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19043.1165]
Chrome (web)      • chrome  • web-javascript • Google Chrome 93.0.4577.63
Edge (web)        • edge    • web-javascript • Microsoft Edge 93.0.961.47
[1]: Windows (windows)
[2]: Chrome (chrome)
[3]: Edge (edge)
Please choose one (To quit, press "q/Q"): 1
Launching lib\main.dart on Windows in debug mode...
../../../flutter/.pub-cache/hosted/pub.dartlang.org/fluent_ui-3.1.0/lib/src/controls/form/selection_controls.dart(50,10): error G76A9B1F6: The method '_FluentTextSelectionControls.buildHandle' has fewer positional arguments than those of overridden method 'TextSelectionControls.buildHandle'. [C:\Users\Elias\Documents\GitHub\subathontimer\build\windows\flutter\flutter_assemble.vcxproj]
../../../flutter/.pub-cache/hosted/pub.dartlang.org/fluent_ui-3.1.0/lib/src/controls/form/selection_controls.dart(61,10): error G76A9B1F6: The method '_FluentTextSelectionControls.getHandleAnchor' has fewer positional arguments than those of overridden method 'TextSelectionControls.getHandleAnchor'. [C:\Users\Elias\Documents\GitHub\subathontimer\build\windows\flutter\flutter_assemble.vcxproj]
C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(241,5): error MSB8066: Custom build for 
'C:\Users\Elias\Documents\GitHub\subathontimer\build\windows\CMakeFiles\46a64e13ad26758e660c9cd0280c46a4\flutter_windows.dll.rule;C:\Users\Elias\Documents\GitHub\subathontimer\build\windows\CMakeFiles\87e0e264a5e31ffa0dc16ab4c2062d92\flutter_assemble.rule' exited with code 1. [C:\Users\Elias\Documents\GitHub\subathontimer\build\windows\flutter\flutter_assemble.vcxproj]
Building Windows application...                                         
Exception: Build process failed.
PS C:\Users\Elias\Documents\GitHub\subathontimer> flutter doctor -v 
[√] Flutter (Channel stable, 2.5.0, on Microsoft Windows [Version 10.0.19043.1165], locale en-US)
    • Flutter version 2.5.0 at C:\Users\Elias\flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 4cc385b4b8 (7 days ago), 2021-09-07 23:01:49 -0700
    • Engine revision f0826da7ef
    • Dart version 2.14.0

[√] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
    • Android SDK at C:\Users\Elias\AppData\Local\Android\sdk
    • Platform android-S, build-tools 30.0.3
    • ANDROID_HOME = C:\Users\Elias\AppData\Local\Android\sdk
    • Java binary at: C:\Program Files\Android\Android Studio\jre\bin\java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[√] Chrome - develop for the web
    • Chrome at C:\Program Files\Google\Chrome\Application\chrome.exe

[√] Visual Studio - develop for Windows (Visual Studio Community 2019 16.11.3)
    • Visual Studio at C:\Program Files (x86)\Microsoft Visual Studio\2019\Community
    • Visual Studio Community 2019 version 16.11.31702.278
    • Windows 10 SDK version 10.0.19041.0

[√] Android Studio (version 2020.3)
    • Android Studio at C:\Program Files\Android\Android Studio
    • Flutter plugin can be installed from:
       https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
       https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[√] VS Code (version 1.60.1)
    • VS Code at C:\Users\Elias\AppData\Local\Programs\Microsoft VS Code
    • Flutter extension version 3.26.0

[√] Connected device (3 available)
    • Windows (desktop) • windows • windows-x64    • Microsoft Windows [Version 10.0.19043.1165]
    • Chrome (web)      • chrome  • web-javascript • Google Chrome 93.0.4577.63
    • Edge (web)        • edge    • web-javascript • Microsoft Edge 93.0.961.47

• No issues found!
```

--- a/lib/src/controls/form/selection_controls.dart
+++ b/lib/src/controls/form/selection_controls.dart
@@ -52,13 +52,20 @@ class _FluentTextSelectionControls extends TextSelectionControls {
     TextSelectionHandleType type,
     double textLineHeight, [
     VoidCallback? onTap,
+    double? startGlyphHeight,
+    double? endGlyphHeight,
   ]) {
     return const SizedBox.shrink();
   }
 
   /// Gets the position for the text selection handles, but desktop has none.
   @override
-  Offset getHandleAnchor(TextSelectionHandleType type, double textLineHeight) {
+  Offset getHandleAnchor(
+    TextSelectionHandleType type,
+    double textLineHeight, [
+    double? startGlyphHeight,
+    double? endGlyphHeight,
+  ]) {
     return Offset.zero;
   }
 

